app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError()
+
+        if not visitor.get("wearing_a_mask", False):

```

Comment:
It's a good practice to explicitly handle the case where the 'wearing_a_mask' key does not exist in the 'visitor' dictionary. Although `visitor.get('wearing_a_mask', False)` will return `False` if the 'wearing_a_mask' key does not exist in the 'visitor' dictionary, it would be clearer to handle this case explicitly: `if 'wearing_a_mask' not in visitor or not visitor['wearing_a_mask']:`.